src/XGBoostSharp/lib/DMatrix.cs

```diff
@@ -68,6 +68,51 @@ void SetFloatInfo(string field, float[] floatInfo)
         ThrowIfError(output);
     }
 
+    public void SetFeatureNames(string[] featureNames)
+    {
+        SetFeatureInfo(featureNames, "feature_name");
```

Comment:
@madskonradsen Let's move the hardcoded strings to a `Fields` class, so we only define them in one place. like this:

```csharp
namespace XGBoostSharp.lib;

public static class Fields
{
#pragma warning disable IDE1006 // Naming Styles
    public const string label = nameof(label);
    public const string feature_name = nameof(feature_name);
    public const string feature_type = nameof(feature_type);
#pragma warning restore IDE1006 // Naming Styles
}
```